packages/wonder-blocks-button/components/button-core.js

```diff
@@ -3,10 +3,14 @@ import React from "react";
 import {StyleSheet} from "aphrodite";
 import {Link} from "react-router-dom";
 
-import {LabelLarge, LabelSmall} from "wonder-blocks-typography";
-import Color, {SemanticColor, mix, fade} from "wonder-blocks-color";
-import {addStyle} from "wonder-blocks-core";
-import type {ClickableHandlers} from "wonder-blocks-core";
+import {LabelLarge, LabelSmall} from "@khanacademy/wonder-blocks-typography";
```

Comment:
What commands did you run to do these replacements? Might be useful for the future!

Reply:
I did a simple find and replace.  I replaced `"wonder-blocks-` with `"@khanacademy/wonder-blocks-`.